src/views/ListOfBooks.jsx

```diff
@@ -37,8 +38,10 @@ const ListOfBooks = () => {
       {books && books.map((book) => {
         return (
           <div className="books-container" key={book.id}>
-            <h3 className={book.valid ? "" : "not-valid-books-title"}>{book.displayTitle}</h3>
-            <img className={book.valid ? "valid-books-img" : "not-valid-books-img"} src={book.url} alt={book.displayTitle} />
+            <Link to={book.valid && `/book/${book.id}`} state={{ book }}>
```

Comment:
It's not clear from the React Router docs whether `<Link to={false} />` is supposed to work.

This may help https://github.com/remix-run/react-router/issues/1082